<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>forgotPasscode</class>
 <widget class="QWidget" name="forgotPasscode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>forgotPasscode</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_title">
     <property name="minimumSize">
      <size>
       <width>261</width>
       <height>71</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>261</width>
       <height>71</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Choose your option.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btn_send">
     <property name="minimumSize">
      <size>
       <width>281</width>
       <height>121</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Send reset code to email</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btn_guest">
     <property name="minimumSize">
      <size>
       <width>281</width>
       <height>121</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Use scale as guest</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>215</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="btn_back">
     <property name="minimumSize">
      <size>
       <width>281</width>
       <height>121</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Back to login</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
